x/auth/types/client_tx.go

```diff
@@ -1,16 +1,18 @@
 package types
 
 import (
-	"github.com/tendermint/tendermint/crypto"
-
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/legacy"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx/signing"
+	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
 )
 
 // StdTxBuilder wraps StdTx to implement to the context.TxBuilder interface
 type StdTxBuilder struct {
 	StdTx
+	cdc *codec.Codec
```

Comment:
why an amino codec?

Reply:
Because `StdTx` is only retained for amino backwards compatibility. The new proto `Tx` types will use `Marshaler`.

Reply:
I see. Can you add that ^^ as a comment on `StdTxBuilder`?

Reply:
Sure

Reply:
👍 adding a deprecation comment and possibly a `Legacy` prefix too. Up to you

Reply:
Marked as deprecated

Reply:
Actually marking it as deprecated causes a ton of linter errors that I don't want to have to deal with in this PR. Instead I updated the godoc comment to explain that the protobuf Tx is preferred.

Is that okay rather than forcing a larger refactor in this PR?